setup/baseConfig.yaml

```diff
@@ -167,7 +167,10 @@ azstorage:
   max-results-for-list: <maximum number of results returned in a single list API call while getting file attributes. Default - 2>
   telemetry : <additional information that customer want to push in user-agent>
   honour-acl: true|false <honour ACLs on files and directories when mounted using MSI Auth and object-ID is provided in config>
-  
+  cpk-enabled: true|false <enable client provided key encryption>
```

Comment:
In README.md file there are sections which lists the env variables and cli parameters. Kindly mention the new parameters and env variables in respective sections.